test/gpu/native/llvmMoved.chpl

```diff
@@ -54,7 +48,7 @@
 
 // After addressing the checklist items (or creating a GH issue to do so)
 // modify the following const so that this test no longer fails.
-const EXPECTED_LLVM_VERSION = 15;
+const EXPECTED_LLVM_VERSION = 16;
```

Comment:
I'm assuming you have talked to GPU team members and/or made the issue here. If not, please do those things.

Reply:
I did address the items in this checklist, but based on what we do with GPU support with LLVM 16 for now in general will change what I need to do here. I may revert these changes in this PR

Reply:
I have reverted these changes, and added a check for gpu+llvm 16